modules/ROOT/pages/platforms.adoc

```diff
@@ -31,6 +31,13 @@ The currently supported platforms and their identifiers are listed below.
 * QEMU (`qemu`): Hypervisor. See xref:provisioning-libvirt.adoc[Booting on libvirt]
 * OpenStack (cloud platform): `openstack`): Cloud platform. See xref:provisioning-openstack.adoc[Booting on OpenStack].
 
+=== s390x
+
+* IBM Cloud, VPC Generation 2 (`ibmcloud`): Cloud platform. See xref:provisioning-ibmcloud.adoc[Booting on IBM Cloud].
+* Bare metal (`metal`): With BIOS, or network boot, with standard or 4k Native disks. See xref:bare-metal.adoc[Installing on Bare Metal] or xref:live-booting-ipxe.adoc[Live-booting via iPXE].
```

Comment:
I'll need some help on this line:

```
With BIOS, or network boot, with standard or 4k Native disks.
```

I don't know if BIOS is the right term. I don't know about network boot. Also don't think 4kNative disks work.. At least we have some tests disabled for them on s390x.

Reply:
Network boot is supported, we have tests for it. For BIOS, maybe just "From disk".
Let's just drop the 4k bit since we're not testing it anyway.